app/src/main/res/values/dimens.xml

```diff
@@ -440,6 +440,11 @@
   <dimen name="all_topics_text_margin_end">32dp</dimen>
   <dimen name="all_topics_text_margin_bottom">12dp</dimen>
 
+  <!-- Profile Chooser Fragment -->
+  <dimen name="profile_chooser_fragment_profile_select_text_marginTop">32dp</dimen>
```

Comment:
Rename to `profile_chooser_fragment_profile_select_text_margin_top`

Same for all other values created in this PR

Reply:
Done